modules/021-cni-cilium/openapi/values.yaml

```diff
@@ -27,6 +27,14 @@ properties:
 
           * - `Netfilter` - use kernel Netfilter(iptables/nf_tables).
           * - `BPF` - use cilium BPF.
+      vmCIDRs:
+        type: array
+        description: |
+          List of CIDRs used to allocate static IP addresses for Virtual Machines.
```

Comment:
refer to the virtualization module pls.

Reply:
And describe, pls, how this discovered arg will be used by cilium.

Reply:
sure, done

Reply:
sure done